### PR TITLE
[SC-596] Add the token bridge + gov relay to base

### PR DESCRIPTION
**Pull request by hexonaut**
This will initialize the token bridge and use those tokens for the psm.

### CODE REVIEW

File: script/input/1/base.json [hunk 1]
```diff
@@ -1,3 +1,4 @@
 {
-  "usdc": "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913"
+  "usdc": "0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913",
+  "clPrefix": "BASE"
```
Comment (krzkaczor) [inline script/input/1/base.json]: what does `clPrefix` stand for?

File: script/SetupAll.s.sol [hunk 2]
```diff
@@ -304,7 +330,111 @@ contract SetupAll is Script {
...
+        ScriptTools.switchOwner(_nst, _deployer, _owner);
+
+        return Nst(_nst);
+    }
+
+    function setupOpStackTokenBridge(OpStackForeignDomain storage domain) internal {
+        address l1CrossDomain;
+        if (domain.name.eq("base")) {
+            l1CrossDomain = OptimismForwarder.L1_CROSS_DOMAIN_BASE;
+        }
```
Comment (krzkaczor) [inline script/SetupAll.s.sol:352]: fail on else to easily spot misconfiguration?

**Review by krzkaczor — APPROVED** [review]

File: script/input/1/base.json (hunk 1 above)
Comment (hexonaut) [reply script/input/1/base.json]: chainlog prefix - can make it more clear

Comment (krzkaczor) [reply script/input/1/base.json]: for me it wasn't clear at all but maybe i am just not familiar with the nomenclature. @lucas-manuel @barrutko thoguhts?

Comment (hexonaut) [reply script/input/1/base.json]: fixed

File: script/SetupAll.s.sol (hunk 2 above)
Comment (hexonaut) [reply script/SetupAll.s.sol:352]: fixed

**Pull request merged**